ocean_lib/example_config.py

```diff
@@ -5,78 +5,120 @@
 
 import logging
 import os
-from typing import Dict, Optional
+import copy
+
+from web3 import Web3
+
+from ocean_lib.config import (
+    Config,
+    config_defaults,
+    SECTION_ETH_NETWORK,
+    NAME_CHAIN_ID,
+    SECTION_RESOURCES,
+    NAME_NETWORK_URL,
+    NAME_PROVIDER_URL,
+    NAME_METADATA_CACHE_URI,
+    DEFAULT_METADATA_CACHE_URI,
+    DEFAULT_PROVIDER_URL,
+)
+from ocean_lib.ocean.util import get_web3_connection_provider
 
 from enforce_typing import enforce_types
-from ocean_lib.config import Config
-from ocean_lib.ocean.util import get_infura_id, get_infura_url
 
 logging.basicConfig(level=logging.INFO)
 
+NETWORK_NAME = "network_name"
+NAME_STORE_INTERVAL = "store_interval"
 
-@enforce_types
-class ExampleConfig:
-    @staticmethod
-    def get_config_net() -> str:
-        """
-        :return: value of environment variable `TEST_NET` or default `ganache`
-        """
-        return os.environ.get("TEST_NET", "ganache")
+CONFIG_NETWORK_HELPER = {
+    1: {
+        NAME_STORE_INTERVAL: 6.5,
+        NAME_PROVIDER_URL: "https://provider.mainnet.oceanprotocol.com",
+        NAME_METADATA_CACHE_URI: "https://aquarius.oceanprotocol.com",
+        NETWORK_NAME: "mainnet",
+    },
+    3: {
+        NAME_STORE_INTERVAL: 6,
+        NAME_PROVIDER_URL: "https://provider.ropsten.oceanprotocol.com",
+        NAME_METADATA_CACHE_URI: "https://aquarius.oceanprotocol.com",
+        NETWORK_NAME: "ropsten",
+    },
+    4: {
+        NAME_STORE_INTERVAL: 7.5,
+        NAME_PROVIDER_URL: "https://provider.rinkeby.oceanprotocol.com",
+        NAME_METADATA_CACHE_URI: "https://aquarius.oceanprotocol.com",
+        NETWORK_NAME: "rinkeby",
+    },
+    56: {
+        NAME_STORE_INTERVAL: 1.5,
+        NAME_PROVIDER_URL: "https://provider.bsc.oceanprotocol.com",
+        NAME_METADATA_CACHE_URI: "https://aquarius.oceanprotocol.com",
+        NETWORK_NAME: "bsc",
+    },
+    137: {
+        NAME_STORE_INTERVAL: 1,
+        NAME_PROVIDER_URL: "https://provider.polygon.oceanprotocol.com",
+        NAME_METADATA_CACHE_URI: "https://aquarius.oceanprotocol.com",
+        NETWORK_NAME: "polygon",
+    },
+    1337: {
+        NAME_STORE_INTERVAL: 2.5,
+        NAME_PROVIDER_URL: DEFAULT_PROVIDER_URL,
+        NAME_METADATA_CACHE_URI: DEFAULT_METADATA_CACHE_URI,
+        NETWORK_NAME: "ganache",
+    },
+}
 
-    @staticmethod
-    def get_base_config() -> Dict[str, Dict[str, str]]:
-        """
-        :return: dict
-        """
-        return {
-            "eth-network": {"network": "http://localhost:8545", "address.file": ""},
-            "resources": {
-                "metadata_cache_uri": "http://aquarius:5000",
-                "provider.url": "http://localhost:8030",
-                "provider.address": "0x068Ed00cF0441e4829D9784fCBe7b9e26D4BD8d0",
-                "downloads.path": "consume-downloads",
-            },
+
+def get_config_helper_network(network_url: str) -> dict:
```

Comment:
I used the parameter `network_url` for `get_config-helper_network`, because I needed to add the network URL in the config dict, but I think I will add in the `get_config` function using `dict.update()` and implement your idea instead.